### PR TITLE
Add styles to the bookstore app

### PR DESCRIPTION
**Pull request by Kaweesi-Matia**
## Below is the description of what is included in the pull request.

- [ ] Add missing JSX elements to the project with reference to the Zeplin design.
- [ ] Add styles to match Zeplin's design.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+.d-flex li {
+  font-size: 14px;
+  font-family: Montserrat, serif;
+}
+
+/* .navigation_container {
+  background: #fff;
+  border-bottom: 1px solid #e8e8e8;
+} */
+
```
Comment (mckent05) [inline src/App.css]: - Please remove all commented code blocks from your code. This would help keep your code clean and make it easy to maintain 👍 👍

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Kaweesi-Matia,

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR looks professional 👏 
- Your project is impressive 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 1 above)
Comment (Kaweesi-Matia) [reply src/App.css]: Remove commented code

**Review by Deepakdanger — APPROVED** [review]
Hi @Kaweesi-Matia, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**